plugin/iro.vim

```diff
@@ -2,11 +2,17 @@ if $IRO_DISABLED
   let g:iro#enabled_filetypes = {'ruby': 0, 'python': 0, 'yaml': 0}
 endif
 
+if exists('g:loaded_iro') || v:version < 800
```

Comment:
I thnks it is more user-friendly if it displays a message to notify too old version when `v:version < 800`.
e.g. `echoerr "iro.vim: This plugin requires Vim 8.0 or newer."`

What do you think?

Reply:
That certainly seems to be better.
We will fix it soon.

Reply:
I fixed. Please check this changing.